bots/pr/src/main/java/org/openjdk/skara/bots/pr/JEPCommand.java

```diff
@@ -55,23 +55,39 @@ private void showHelp(PrintWriter reply) {
                  * `/jep 1234567`
                  * `/jep jep-123`
                  * `/jep JEP-123`
+                 * `/jep 123`
```

Comment:
I would suggest putting this example first.

Also, there is an empty line after "Some examples:" but not after "Command syntax:". That looks weird.

Reply:
Fixed.